README.md

```diff
@@ -269,6 +269,82 @@ mv s3://bucket/2020/03/18/file1.gz s3://bucket/2020/03/18/original/file.gz
 ls # inline comments are OK too
 ```
 
+#### Sync
+With `sync` command, it's possible to synchronise files between an S3 bucket and a local directory;
+
+Suppose we have following files;
```

Comment:
Before diving into how it works, let's make it clear that we never touch anything in the source. We never delete anything from the source. Sync code assumes the source as the truth.

Reply:
Tried to make it clear with a little bit of glimpse on how it syncs. How does it sound like?

Reply:
Looks better. Thanks.